package.json

```diff
@@ -35,7 +35,7 @@
     "prepublish": "npm run build"
   },
   "dependencies": {
-    "immutable": "3.7.0"
+    "immutable": ">=3.8.1"
```

Comment:
Should we be using peerDependencies for this? It kinda seems like it but I'm not entirely sure... any thoughts?